src/udp.rs

```diff
@@ -14,3 +14,22 @@ pub struct UdpHdr {
 impl UdpHdr {
     pub const LEN: usize = mem::size_of::<UdpHdr>();
 }
+
+#[test]
+#[cfg(feature = "serde")]
+#[cfg(feature = "bincode")]
```

Comment:
This is not going to work? There is no "bincode" feature. You just added bincode as a dev-dependency.

I think you could just write the test in the following way:

```rust
#[cfg(test)]
mod test {
    #[cfg(feature = "serde")]
    #[test]
    fn test_serialize() {
        // here goes your code
    }
}
```

Reply:
It did work, thanks to the `dep:bincode` legacy notation :)
---
Done.